src/support/cleanse.h

```diff
@@ -8,6 +8,8 @@
 
 #include <stdlib.h>
 
+// Securely erase a span of memory in a way guaranteed not to be
```

Comment:
On further research, I'm not sure OPENSSL_cleanse actually guarantees anything at all.